tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousTest.java

```diff
@@ -101,33 +91,31 @@ public void testAsyncIsFinished() {
      */
     @Test
     public void testClassLevelAsyncIsNotFinished() {
-        Future<Connection> future = null;
-        try {
-            future = clientClass.service();
-        }
-        catch (InterruptedException e) {
-            throw new AssertionError("testAsync: unexpected InterruptedException calling service");
-        }
-
-        Assert.assertFalse(future.isDone());
-
+        CompletableFuture<Void> waitingFuture = newWaitingFuture();
+        CompletionStage<Connection> resultFuture = clientClass.serviceCS(waitingFuture);
```

Comment:
The original test called `service()`, so this should also call `service()`.

Reply:
done